### PR TITLE
Handling new clinical dataset from UW

### PR DESCRIPTION
**Pull request by tsibley**
Explanations in the commit messages.

This is important to do by tomorrow so we can load it and then Mike can re-run his models by the end of the week and then we can deploy the current conditions map.

### CODE REVIEW

File: lib/seattleflu/id3c/cli/command/clinical.py [hunk 1]
```diff
@@ -117,91 +114,13 @@ def parse_uw(uw_filename, uw_nwh_file, hmc_sch_file, output):
     dump_ndjson(clinical_records)
 
 
-def load_data(uw_filename: str, uw_nwh_file: str, hmc_sch_file: str):
```
Comment (kairstenfay) [inline lib/seattleflu/id3c/cli/command/clinical.py]: I'm so glad to see all those functions go.

File: lib/seattleflu/id3c/cli/command/clinical.py [hunk 2]
```diff
@@ -80,7 +80,6 @@ def parse_uw(uw_filename, output):
         'Collection_ID': 'barcode',
         'EthnicGroup': 'HispanicLatino',
         'Fac': 'site',
-        'FinClass': 'MedicalInsurance',
```
Comment (kairstenfay) [inline lib/seattleflu/id3c/cli/command/clinical.py]: I made a Trello card for this.

File: clinical/uw/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+  a. be replaced by direct usage of geocoding and location lookup
+     functions in `id3c clinical parse-uw`, or
+
+  b. grow and take over more parts of `id3c clinical parse-uw`,
+     eventually replacing it with a pipeline of `geocode`, `location
+     lookup`, `de-identify` and other (not yet existing) core commands.
+
+I think (a) is the more sensible option in the short and middle term.  Option
+(b) might be called the "augur strategy", which might be better longer term.
```
Comment (kairstenfay) [inline clinical/uw/README.md]: Thank you for documenting the work done here and future directions.

File: lib/seattleflu/id3c/cli/command/clinical.py (hunk 2 above)
Comment (tsibley) [reply lib/seattleflu/id3c/cli/command/clinical.py]: Thank you!

**Comment by tsibley** [comment]
Deployed and data uploaded.

**Pull request merged**